piclassifier/locationconfig.py

```diff
@@ -0,0 +1,51 @@
+from pathlib import Path
+import attr
+import yaml
+
+CONFIG_FILENAME = "location.yaml"
+CONFIG_DIRS = [Path(__file__).parent.parent, Path("/etc/cacophony")]
+
+
+@attr.s
+class LocationConfig:
+
+    DEFAULT_LAT = -43.5321
+    DEFAULT_LONG = 172.6362
```

Comment:
Don't set a default location if it's missing!

The only time it's ok to use a default location is for sunset/sunrise calculations - not for recordings.